Tools/Scripts/hooks/pre-push

```diff
@@ -215,6 +215,18 @@ def security_level_for_commit(commit, remotes, name_to_remote=None):
     message = message_for(commit)
     title = message.splitlines()[0]
 
+    # Compute class 3 before class 2, because a class 3 violation is more interesting than a class 2
```

Comment:
This makes me think the classes should have semantically meaningful names, since they aren’t even being checked in escalating order. No need to fix now unless you want to, just my 2c.

Reply:
I agree, going to put that in a different change, though, because it will be a larger change.